openeo/UDF.ipynb

```diff
@@ -230,9 +230,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "OpenEO",
+   "display_name": "openeo_py",
    "language": "python",
-   "name": "openeo"
```

Comment:
Note that changing the kernel name might trigger a "Select kernel" popup to users that open this notebook on CDSE jupyterlab when the `name` does not exist there.

In this case kernel `python3`  does exist on CDSE jupyterlab, but it's the base python and  doesn't have any of the importabt packages, like  `openeo` installed I think

Reply:
Ahh sorry for confusion @soxofaan. Changes were made locally and didn't realise that change in markdown could also trigger it

Reply:
apparently this is documented here: https://github.com/eu-cdse/notebook-samples#kernel-spec-metadata

Reply:
yes updated

Reply:
FYI: under #22 I added a kernel check script that now runs as pre-commit and github actions workflow